tests/types/email.spec.ts

```diff
@@ -0,0 +1,74 @@
+import {StrongEmail, makeEmail} from '../../src/types/email';
+
+const MOCK_INITIAL = 'test@test.com';
+const MOCK_FALLBACK_DEFAULT = 'dog@cat.com';
+const MOCK_FALLBACK = 'toy@undertree,com';
+
+describe('StrongEmail', () => {
+	describe('Implementation', () => {
+		it('should set initial value to sampleInitial argument', () => {
+			const sampleInitial = 'fall@leaves.com';
```

Comment:
Not a requested change, just wanted to say I liked `fall@leaves.com`. Made me laugh.